.gitignore

```diff
@@ -1,14 +1,37 @@
 .DS_Store
-
 .idea/
+backend/.DS_Store
+backend/.idea/
 
-.gradle
-/build/
+backend/.gradle
+backend/build/
 # Ignore Gradle GUI config
-gradle-app.setting
+backend/gradle-app.setting
 
 # Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
-!gradle-wrapper.jar
+!backend/gradle-wrapper.jar
 
 # Cache of project
-.gradletasknamecache
\ No newline at end of file
+backend/.gradletasknamecache
+
+# dependencies
+frontend/node_modules
+frontend/.pnp
+frontend/.pnp.js
+
+# testing
+frontend/coverage
+
+# production
+frontend/build
+
+# misc
+frontend/.DS_Store
```

Comment:
.DS_Store can be global. In fact you can even set that so it's never checked in on your machine.

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

I use that for things like ~.DS_Store~ and emacs stuff.

Reply:
Great thanks